src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -230,49 +230,10 @@
           </v-col>
         </v-row>
 
-        <v-expand-transition>
-          <!-- Upload Affidavit -->
-          <v-row
-            v-if="isContinuationInAffidavitRequired"
-            class="mt-6"
-            no-gutters
-          >
-            <v-col
-              cols="12"
-              sm="3"
-            >
-              <label>Upload Affidavit</label>
-            </v-col>
-
-            <v-col
-              cols="12"
-              sm="9"
-            >
-              <p>
-                Upload the affidavit from the directors.
-              </p>
-
-              <ul>
-                <li>Use a white background and a legible font with contrasting font colour</li>
-                <li>PDF file type (maximum 30 MB file size)</li>
-              </ul>
-
-              <FileUploadPreview
-                class="mt-4"
-                inputFileLabel="Affidavit from directors"
-                :maxSize="MAX_FILE_SIZE"
-                :pdfPageSize="PdfPageSize.LETTER_SIZE"
-                :hint="business.affidavitFile ? 'File uploaded' : undefined"
-                :inputFile="business.affidavitFile"
-                :showErrors="getShowErrors"
-                :customErrorMessage.sync="customErrorMessage"
-                :isRequired="getShowErrors"
-                @fileValidity="onFileValidity($event)"
-                @fileSelected="onFileSelected($event)"
-              />
-            </v-col>
-          </v-row>
-        </v-expand-transition>
+        <!-- Upload Affidavit -->
+        <UploadAffidavit
+          :business="business"
+        />
```

Comment:
I'd like you to keep the v-row and v-col in here since it's part of the layout in this component.

If you keep the v-row and v-col in the sub-component then that's how it must be used in the future... but we might want it in a different layout. So, remove it from there.

The principle I'm following here is scope from a layout point of view.

Reply:
Got it! Updated